### PR TITLE
Conservations checks

### PR DESCRIPTION
**Pull request by NNSSA**

### CODE REVIEW

File: tests/conservation_sum_rules/__main__.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+interactions_pre = (
+	SMI.neutrino_interactions(leptons=[electron], neutrinos=[neutrino_e, neutrino_mu, neutrino_tau])
+)
+
+# for-loop below gets rid of non-elastic processes.
+# For some reason operators as del/remove/pop don't work properly here. This is the only way I managed to make it work.
+# And yes, I know, it looks terrible.
+for x in interactions_pre:
+    for n, y in enumerate(x.integrals):        
+        if (y.reaction[0].specie.symbol != y.reaction[2].specie.symbol and y.reaction[0].specie.symbol != y.reaction[3].specie.symbol):
```
Comment (ckald) [inline tests/conservation_sum_rules/__main__.py]: Your check is not very good. Indeed, this is how you can roughly filter some of the integrals you don't want to see, but you missed the following:

Every reaction creates multiple integrals. `A + B -> C + D` creates 4 terms in 4 different Boltzmann equations. `A + B -> A + B` will produce 2: `I_A(A+B->A+B)` and `I_B(B+A->A+B)`. The general notation is that the first particle in the integral is the one corresponding to Boltzmann equations. Others are ordered by species for convenience.

I would make a check like this:

```
from collections import Counter

left_species = Counter(item.specie for item in integral.reaction if item.side == -1)
right_species = Counter(item.specie for item in integral.reaction if item.side == 1)

if left_species == right_species:
```

Also, one-letter variable names produce lots of bugs. Python is designed such that you really can program without counters `i, j, k, l, ...` familiar from C. So there is no reason to call an `interaction` `x`

**Comment by ckald** [comment]
You've committed a bunch of long log files. They will be overwritten after every run, then they would be committed again, and again and again... I decided long ago that it is best not to put them into the repo. The alternative would be to separate the runs and the source, but it does not seem to be the right thing here right now — I could even make a formal Python package out of pyBBN, but it would increase the learning curve and might complicate development process. I think it's best to postpone that.

Please remove txt files using `git rm <filename>`

**Comment by NNSSA** [comment]
Thought it would be useful for once, in case you want to use them yourself. 
But I will remove them.

**Comment by ckald** [comment]
It does not take astronomical time to run the code, so I would rather run it again :)

**Comment by NNSSA** [comment]
Didn't really know how to make it empty...

**Pull request merged**